api/src/pcapi/routes/backoffice/templates/venue/get.html

```diff
@@ -274,6 +274,18 @@ <h5 class="modal-title">Supprimer le lieu {{ venue.common_name }}</h5>
                   {{ venue.criteria | format_criteria | safe }}
                 </p>
               {% endif %}
+              {% if venue.accessibilityProvider %}
+                <p class="mb-1">
+                  <span class="fw-bold">Lien vers la page Acceslibre :</span>
+                  {% if venue.accessibilityProvider.externalAccessibilityUrl %}
+                    <a href="{{ venue.accessibilityProvider.externalAccessibilityUrl | format_website }}"
+                       target="_blank"
+                       class="link-primary">{{ venue.accessibilityProvider.externalAccessibilityUrl | format_website }}</a>
+                  {% else %}
+                    Attention, l'URL pour l'identifiant acceslibre <i>{{ venue.accessibilityProvider.externalAccessibilityId }}</i> n'est pas renseigné
```

Comment:
```suggestion
                    Attention, l'URL pour l'identifiant acceslibre <i>{{ venue.accessibilityProvider.externalAccessibilityId }}</i> n'est pas renseignée
```
Question subsidiaire : qu'est-ce qui indique qu'on a bien un `externalAccessibilityId`, puisqu'il est `nullable` ?